src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/AbstractExpressionHandler.java

```diff
@@ -319,7 +319,7 @@ private void checkLineIndent(DetailAST ast,
         final int start = getLineStart(line);
         final int columnNumber = expandedTabsColumnNo(ast);
         // if must match is set, it is a violation if the line start is not
-        // at the correct indention level; otherwise, it is an only an
+        // at the correct indention level; otherwise, it is the only
         // violation if this statement starts the line and it is less than
```

Comment:
Should this be `it is only a violation if`.... ?

Reply:
`it is only a violation if .... `  sounds more correct to me too

Reply:
done